cx-portal/src/components/pages/AppSubscription/AppSubscriptionDetailOverlay/index.tsx

```diff
@@ -0,0 +1,166 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 BMW Group AG
+ * Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+import { useTranslation } from 'react-i18next'
+import {
+  Dialog,
+  DialogContent,
+  DialogHeader,
+  StaticTable,
+  TableType,
+  Typography,
+} from 'cx-portal-shared-components'
+import { useFetchSubscriptionDetailQuery } from 'features/appSubscription/appSubscriptionApiSlice'
+import ReleaseStepper from 'components/shared/basic/ReleaseProcess/stepper'
+import { SubscriptionStatus } from 'features/apps/apiSlice'
+
+interface AppSubscriptionDetailProps {
+  openDialog: boolean
+  appId: string
+  subscriptionId: string
+  handleOverlayClose: () => void
+}
+
+const AppSubscriptionDetailOverlay = ({
+  openDialog = false,
+  appId,
+  subscriptionId,
+  handleOverlayClose,
+}: AppSubscriptionDetailProps) => {
+  const { t } = useTranslation()
+  const { data } = useFetchSubscriptionDetailQuery({ appId, subscriptionId })
+
+  const stepLists = [
+    {
+      headline: t('content.appSubscription.detailOverlay.stepLists.firstStep'),
+      step: 1,
+    },
+    {
+      headline: t('content.appSubscription.detailOverlay.stepLists.secondStep'),
+      step: 2,
+    },
+    {
+      headline: t('content.appSubscription.detailOverlay.stepLists.thirdStep'),
+      step: 3,
+    },
+  ]
+
+  const subscriptionDetails: TableType = {
+    head: [t('content.appSubscription.detailOverlay.subscriptionDetails'), ''],
+    body: [
+      [
+        `${t('content.appSubscription.detailOverlay.appTitle')}`,
```

Comment:
no need to wrap in a template string:
`t('...'),` 
should be okay. Also for following lines.

Reply:
@oyo It's accepting string only but not accepting translation string. So I did in this way.